files/docker-image-prune.service

```diff
@@ -0,0 +1,11 @@
+# SPDX-FileCopyrightText: 2025 Stefan Haun <tux@netz39.de>
+# SPDX-License-Identifier: MIT
+
+[Unit]
+Description=Prune unused Docker images
+Requires=network-online.target docker.service
+After=network-online.target docker.service
```

Comment:
Why is network required here?

Reply:
Good question! Should not be needed here. I cannot really test, but I assume having Docker is sufficient.